pipelines/DP1Paper.yaml

```diff
@@ -52,12 +52,25 @@ tasks:
     class: lsst.analysis.tools.tasks.PerTractPropertyMapAnalysisTask
     config:
       connections.outputName: propertyMapTract
-      zoomFactors: [2, 8]
       projectionKwargs:
         celestial: true
         gridlines: true
+        min_lon_ticklabel_delta: 0.12
+      publicationStyle: true
       # The entries in the 'atools' namespace must exactly match the dataset types.
       atools.deepCoadd_dcr_ddec_map_weighted_mean: PerTractPropertyMapTool
-      publicationStyle: true
+      atools.deepCoadd_dcr_dra_map_weighted_mean: PerTractPropertyMapTool
+      atools.deepCoadd_dcr_e1_map_weighted_mean: PerTractPropertyMapTool
```

Comment:
This pipeline is supposed to be an example pipeline and doesn't need all of the property maps, just pick your favorite.